### PR TITLE
DB operator tests with Istio enabled WebLogic domains

### PR DESCRIPTION
**Pull request by sankarpn**
Add integration tests for new istio enabled weblogic domain and JRF domain using databse instance created using DB operator.

Jenkins result:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7707
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7705
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7704
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7703

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java [hunk 1]
```diff
@@ -0,0 +1,836 @@
...
+  private static final String RCUSCHEMAPREFIX = "FMWDOMAINMII";
+  private static final String RCUSYSPASSWORD = "Oradoc_db1";
+  private static final String RCUSCHEMAPASSWORD = "Oradoc_db1";
+  private static final String modelFile = "model-singleclusterdomain-sampleapp-jrf.yaml";
+
+  private static String dbUrl = null;
+  private static String dbName = "my-oracle-sidb";
+  private static LoggingFacade logger = null;
+
+  private String fmwDomainUid = "fmwdomain-mii-db";
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java:125]: I want the domainuid to be unique across all the class for better triaging prospective.
$ grep fmwdomain-mii-db *.java
ItDBOperator.java:  private String fmwDomainUid = "fmwdomain-mii-db";

Can u make the domaiuid to be unique such as 'fmwdomain-istio-mii-db"

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java [hunk 2]
```diff
@@ -0,0 +1,836 @@
...
+  private String fmwManagedServerPrefix = fmwDomainUid + "-managed-server";
+  private int replicaCount = 2;
+  private String fmwAminSecretName = fmwDomainUid + "-weblogic-credentials";
+  private String fmwEncryptionSecretName = fmwDomainUid + "-encryptionsecret";
+  private String rcuaccessSecretName = fmwDomainUid + "-rcu-access";
+  private String opsswalletpassSecretName = fmwDomainUid + "-opss-wallet-password-secret";
+  private String opsswalletfileSecretName = fmwDomainUid + "opss-wallet-file-secret";
+  private String adminSvcExtHost = null;
+
+  private static final String wlsDomainUid = "mii-jms-recovery-db";
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java:136]: Make the wlsDomainUid unique mii-jms-istio-recovery-db

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java]: fixed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java (hunk 2 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDBOperator.java]: fixed

**Review by anpanigr — COMMENTED** [review]
Publish the kind new results with sync from main branch.

**Review by ddsharpe — CHANGES_REQUESTED** [review]
Update copyright dates to 2022

**Comment by robertpatrick** [comment]
What's going on with this PR?  @maggiehe00 @anpanigr @bhavaniravichandran, please review.

**Comment by bhavaniravichandran** [comment]
Can you please run FMW tests and other tests using db in OKD env and upload the results?

**Review by anpanigr — APPROVED** [review]
lgtm

**Comment by sankarpn** [comment]
> Publish the kind new results with sync from main branch.

done.

**Comment by maggiehe00** [comment]
We can add this test to okd cert profile after ItDBOperator if we decide to do so.  For now we can open a track JIRA first.

**Review by maggiehe00 — APPROVED** [review]

**Review by robertpatrick — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**